services/Sport.js

```diff
@@ -0,0 +1,29 @@
+const SportEntity = require('../entities/Sport');
+const Sport = require('../models/sports');
+
+const getAllSports = async() => {
+    const sportsData = await SportEntity.findAllSports();
+    sportsNames = sportsData.map(sport => sport.label);
+    return sportsNames;
+};
+
+const findSportByName = async(sportName) => {
+    const sportData = await SportEntity.findSportBySportName(sportName);
+    return sportData;
```

Comment:
```suggestion
    return SportEntity.findSportBySportName(sportName);
```
Maybe in an otherpr you can simplify all those functions